src/main/resources/templates/repo/web/acl/uri-query-xss-statement.json

```diff
@@ -0,0 +1,124 @@
+{
+	"AndStatement": {
+		"Statements": [
+			{
+				"OrStatement": {
+					"Statements": [
+						{
+							"XssMatchStatement": {
+								"FieldToMatch": {
+									"UriPath": {
+									}
+								},
+								"TextTransformations": [
+									{
+										"Priority": 0,
+										"Type": "URL_DECODE"
+									}
+								]
+							}
+						},
+						{
+							"XssMatchStatement": {
+								"FieldToMatch": {
+									"QueryString": {
+									}
+								},
+								"TextTransformations": [
+									{
+										"Priority": 0,
+										"Type": "HTML_ENTITY_DECODE"
+									}
+								]
+							}
+						},
+						{
+							"XssMatchStatement": {
+								"FieldToMatch": {
+									"UriPath": {
+									}
+								},
+								"TextTransformations": [
+									{
+										"Priority": 0,
+										"Type": "HTML_ENTITY_DECODE"
+									}
+								]
+							}
+						},
+						{
+							"XssMatchStatement": {
+								"FieldToMatch": {
+									"QueryString": {
+									}
+								},
+								"TextTransformations": [
+									{
+										"Priority": 0,
+										"Type": "URL_DECODE"
+									}
+								]
+							}
+						}
+					]
+				}
+			},
+			{
+				"NotStatement": {
+					"Statement": {
+						"OrStatement": {
+							"Statements": [
+								{
+									"RegexMatchStatement": {
+										"RegexString": "/repo/v1/thread/.*/threadcount",
```

Comment:
We might want to be more specific in the regex (e.g. replace .* with the digits?)